src/content/docs/en/guides/cms/storyblok.mdx

```diff
@@ -443,7 +443,9 @@ const story = data.story;
 </html>
 ```
 
-This file will generate a page for each story, with the slug and content fetched from the Storyblok API. 
+This file will generate a page for each story, with the slug and content fetched from the Storyblok API.
+
+> When adding folders inside of storyblok, you'll want to include them when interacting with the Storyblok API. For example, in the get request above we could use **cdn/stories/blog**, with a blog folder inside rather than using them at the root.
```

Comment:
Don't you think this would function better as a note component?

also improved the wording a bit.

```suggestion
:::note
When adding folders inside of Storyblok, you'll want to include them when interacting with the Storyblok API. For example, in the GET request above we could use **cdn/stories/blog**, with a blog folder inside rather than using them at the root.
:::
```